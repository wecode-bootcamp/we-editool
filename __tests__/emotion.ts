import styled from '@emotion/styled';
import { css } from '@emotion/react';

describe('library: emotion', () => {
  test('css template test', () => {
    const cssTemplate = css`
      display: flex;
    `;
    expect(cssTemplate).toMatchSnapshot();
  });

  test(`styled test`, () => {
    const StyledDiv = styled.div`
      display: flex;
    `;
    expect(StyledDiv).toMatchSnapshot();
  });
});
