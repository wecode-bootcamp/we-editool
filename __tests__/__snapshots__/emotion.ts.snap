// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`library: emotion css template test 1`] = `
Object {
  "map": undefined,
  "name": "vyoujf",
  "next": undefined,
  "styles": "
      display: flex;
    ",
  "toString": [Function],
}
`;

exports[`library: emotion styled test 1`] = `
Object {
  "$$typeof": Symbol(react.forward_ref),
  "__emotion_base": "div",
  "__emotion_forwardProp": undefined,
  "__emotion_real": [Circular],
  "__emotion_styles": Array [
    "
      display: flex;
    ",
  ],
  "defaultProps": undefined,
  "render": [Function],
  "withComponent": [Function],
}
`;
